doc/api/fs.md

```diff
@@ -469,12 +469,57 @@ no effect on Windows (will behave like `fs.constants.F_OK`).
 
 The final argument, `callback`, is a callback function that is invoked with
 a possible error argument. If any of the accessibility checks fail, the error
-argument will be populated. The following example checks if the file
-`/etc/passwd` can be read and written by the current process.
+argument will be populated. 
+
+The following examples explains the use of different fs.constants.
```

Comment:
I think we can remove this sentence too. If you do wish to keep it, please change `explains` to `explain` and surround `fs.constants` with ``` ` ```.

Reply:
Sure. I've removed the text. Any more feedback?